site/content/3.12/develop/http-api/monitoring/logs.md

```diff
@@ -350,99 +356,113 @@ paths:
+                    execution. DB-Servers and Coordinators log the cluster-internal
+                    communication around AQL queries. It also reports the AQL
+                    memory limit on startup.
                   type: string
                 arangosearch:
                   description: |
-                    One of the possible log topics.
+                    Logs information related to ArangoSearch including Analyzers,
+                    the column cache, and the commit and consolidation threads.
                   type: string
                 audit-authentication:
                   description: |
-                    One of the possible log topics (_Enterprise Edition only_).
+                    Controls whether events such as successful logins and
+                    missing or wrong credentials are written to the audit log
+                    (_Enterprise Edition only_).
                   type: string
                 audit-authorization:
                   description: |
-                    One of the possible log topics (_Enterprise Edition only_).
+                    Controls whether events such as users trying to access databases
+                    without the necessary permissions are written to the audit log
+                    (_Enterprise Edition only_).
                   type: string
                 audit-collection:
                   description: |
-                    One of the possible log topics (_Enterprise Edition only_).
+                    Controls whether events about collections creation, truncation,
+                    and deletion are written to the audit log (_Enterprise Edition only_).
                   type: string
                 audit-database:
                   description: |
-                    One of the possible log topics (_Enterprise Edition only_).
+                    Controls whether events about database creation and deletion
+                    are written to the audit log (_Enterprise Edition only_).
                   type: string
                 audit-document:
                   description: |
-                    One of the possible log topics (_Enterprise Edition only_).
+                    Controls whether document read and write events are written
+                    to the audit log (_Enterprise Edition only_).
                   type: string
                 audit-hotbackup:
                   description: |
-                    One of the possible log topics (_Enterprise Edition only_).
+                    Controls whether the Hot Backup creation, restore, and delete
+                    events are written to the audit log (_Enterprise Edition only_).
                   type: string
                 audit-service:
                   description: |
-                    One of the possible log topics (_Enterprise Edition only_).
+                    Unused log topic (_Enterprise Edition only_).
```

Comment:
It is not fully unused. I think there are messages in this topic when the audit-logging starts up at server start. I am not saying they are useful, but we established that this topic is still in use.